### ISSUE TITLE
GURA02 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Roslyn.Asserts#117]
```cs
        [TestCase("int")]
        [TestCase("string")]
        public void EmptyForPrimitiveTypes(string type)
        {
            var before = @"
namespace RoslynSandbox
{
    public class C
    {
        /// <summary>
        /// Summary
        /// </summary>
        public static void M(string ↓text)
        {
        }
    }
}".AssertReplace("string", type);

            var after = @"
namespace RoslynSandbox
{
    public class C
    {
        /// <summary>
        /// Summary
        /// </summary>
        /// <param name=""text""></param>
        public static void M(string text)
        {
        }
    }
}".AssertReplace("string", type);
            RoslynAssert.CodeFix(Analyzer, Fix, before, after);
        }
```

Maybe it is the `AssertReplace()` messing up analysis.
